src/Runner/TestHandler.php

```diff
@@ -113,7 +113,7 @@ private function initiatePhpVersion($version, PhpInterpreter $interpreter)
 
 	private function initiatePhpIni($value, PhpInterpreter $interpreter)
 	{
-		$interpreter->arguments .= ' -d ' . Helpers::escapeArg($value);
+		$interpreter->addArguments(' -d ' . Helpers::escapeArg($value));
```

Comment:
> The space at the beginning looks a bit weird

It looks. As I wrote sooner, it is an internal code. I was thinking about passing arguments as an unescaped array like `addArguments(['-d', 'some value'])`.

> Have you considered moving the logic for arguments handling to interpreters

I did. But it needs `-d something` (passed by user) parsing which I would like to prevent if not needed. And the name would be probably `addPhpIniOption()` since one option can be set multiple times.